### PR TITLE
Update telling-git-about-your-signing-key.md

### PR DESCRIPTION
**Pull request by austenstone**
### Why:

On Mac OS, gpg signing a commit can require a password. This password prompt is not available when committing through an application like VSCode. By adding `pinentry-mac` the user will be prompted to enter the password when necessary. Found at https://github.com/microsoft/vscode/issues/43809.

Windows doesn't seem to have this problem because the pinentry dialog is installed with gpg. When committing in vscode the user will get a password prompt as expected.

This information could also be moved under [Generating a GPG key](https://docs-13825--patch-2.herokuapp.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key#generating-a-gpg-key)

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@austenstone 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Dise007 — COMMENTED** [review]
Nice

**Comment by HUNTINGTONPRIVADO** [comment]
> > isaiarellano70@gmail.com**_[]()@Dise007 ~~~~_**

File: content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ If you have multiple GPG keys, you need to tell Git which one to use.
   $ if [ -r ~/.bash_profile ]; then echo 'export GPG_TTY=$(tty)' >> ~/.bash_profile; \
     else echo 'export GPG_TTY=$(tty)' >> ~/.profile; fi
   ```
+1. Optionally install `pinentry-mac` using [Homebrew](https://brew.sh/) to open PIN or passphrase entry dialogs when necessary:
```
Comment (hubwriter) [inline content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md:47]:
```suggestion
1. Optionally, to prompt you to enter a PIN or passphrase when required, install `pinentry-mac`. For example, using [Homebrew](https://brew.sh/):
```

**Review by hubwriter — APPROVED** [review]
@austenstone - Many thanks for raising this PR to improve our docs.

I've made a very slight rewording change, for style, but I'm going to go ahead and merge/publish this now. 🚀 

Thanks again. 👍

**Pull request merged**